src/app.js

```diff
@@ -1 +1,28 @@
 'use strict';
+/* eslint-disable no-console */
+
+const { terminal } = require('./terminal');
+const fs = require('fs');
+const { isEqual } = require('lodash');
+
+function copyFile(from, to) {
+  if (isEqual(from, to)) {
+    console.warn('You can\'t copy a file in the same path');
+  } else {
+    fs.copyFile(from, to, err => {
+      if (err) {
+        return console.error(err);
+      }
```

Comment:
Try to use [guard clauses](https://medium.com/@timothydan/javascript-guard-clauses-64b999e3240) instead of nested if/else statements